src/yunohost/certificate.py

```diff
@@ -810,9 +810,20 @@ def _domain_is_accessible_through_HTTP(ip, domain):
 
 
 def _name_self_CA():
-    cert = crypto.load_certificate(
-        crypto.FILETYPE_PEM, open(SELF_CA_FILE).read())
-    return cert.get_subject().CN
+    ca_conf = os.path.join(SSL_DIR, "openssl.ca.cnf")
```

Comment:
There is a ```Subject : CN=``` and a ```Issuer : CN=``` whenprinting the cert content.
So if both domains are the same, then it is self-signed ?